FindTown/FindTownCore/Sources/FindTownCore/Base/BaseNavigationController.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  File.swift
+//  
+//
+//  Created by 이호영 on 2022/12/23.
+//
+
+import UIKit
+import FindTownUI
+
+public class BaseNavigationController: UINavigationController {
+
+    public override func viewDidLoad() {
+        super.viewDidLoad()
+
```

Comment:
```suggestion
open class BaseNavigationController: UINavigationController {

    open override func viewDidLoad() {
        super.viewDidLoad()

```
![스크린샷 2022-12-24 오후 11 53 30](https://user-images.githubusercontent.com/50910456/209441479-d19e40fa-0c20-45e6-9574-1a010d7e96a5.png)

public 으로 하면 오버라이드와 서브클래싱이 안돼서 요부분 open 으로 수정해야 할 거 같아요! 😎

Reply:
서브클래싱하는 목적이 아니라서 public으로 두었어요!

TabBarController, NavigationController내 appearance를 지정하기 위한 클래스에요 ☺️